### ISSUE TITLE
[Feature] Display to BBD users if other users are using BBD

### ISSUE THREAD

**Issue by AlexejheroYTB** [rauenzi/BetterDiscordApp#274]
I have recently been thinking of ways to improve user connection on BBD, and I thought about having a way to let BBD users identify each other.

A request could be periodically made by the client to an external BBD server, which would keep track of that user. When another user would click on a user profile/pop-up of the original user, a request could be sent to the server asking if that user is using BBD, in which case a BBD badge could appear.

Obviously there would be caching, both on the server-side and on the client-side, and there would be a setting to disable this.

This is not a feature request, but rather a suggestion. If accepted, I can submit a pull request to add this myself.

I'd like to hear everyone's thoughts on this.

**Comment by Fleepy** [comment]
This would be very nice! But I fear that if the external server is given in every BD release (unless its very obfuscated, then there shouldnt be a problem), Discord can download that release, and look through the code, and find the server address, and depending on how the server is set up, Discord can issue a massive ban wave since it does violate TOS. (which I doubt they would do, but its a possibility) 

Basically, **That would be kind of hard to do**

**Comment by AlexejheroYTB** [comment]
Discord can download the release, look through the code, find BBD-specific code and issue a ban wave even without the external server.

**Comment by Fleepy** [comment]
> Discord can download the release, look through the code, find BBD-specific code and issue a ban wave even without the external server.

Exactly.

**Comment by piks** [comment]
Against this feature as well, as stated above would make it far too easy to detect usage.

**Comment by rauenzi** [comment]
It wouldn't affect Discord's ability to see who uses BD. Discord can easily test for BD users and ban them programmatically--it would be trivial to do so. Fact is, they actively ignore errors involving BetterDiscord, so they clearly don't care to just blanket ban.

Whether this should be a feature or not I'm not sure. I'd have to maintain a server for that and it seems like more hassle than it's worth.

**Comment by ImSkully** [comment]
Also against this feature, there generally really is no reason to provide a feature for users to recognize other BD users.

**Comment by AlexejheroYTB** [comment]
Okay, thanks for the feedback everyone! :+1:
